README.md

```diff
@@ -4,6 +4,7 @@ puppet-splunk_hec
 Breaking Changes
 -----------
 - splunk_hec::url parameter now expects a full URI of https://servername:8088/services/collector
+- Switches to the fact terminus cache setting via routes.yaml to ensure compatibility with CD4PE, see Fact Terminus Support for guides on how to change it. Prior to deploying this module, remove the setting `facts_terminus` from the `puppet_enterprise::profile::master` class in the `PE Master` node group in your environment if you set it in previous revisions of this module. It will prevent PE from operating normally if left on.
```

Comment:
I'd maybe add in a version number for these two breaking changes, just so we can keep track of when they were introduced

Reply:
+1

Called out versions for both of those breaking changes (polished readme conforming to module standards should be 1.0 objective)